# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pcsx2/no-tests.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pcsx2-8e8b9a3cff64ddc43da12951347b009763e96aa/cmake/BuildParameters.cmake.vanilla	2022-01-12 01:01:09.671962239 +0100
+++ pcsx2-8e8b9a3cff64ddc43da12951347b009763e96aa/cmake/BuildParameters.cmake	2022-01-12 01:01:16.233962550 +0100
@@ -16,7 +16,7 @@
 # Misc option
 #-------------------------------------------------------------------------------
 option(DISABLE_BUILD_DATE "Disable including the binary compile date")
-option(ENABLE_TESTS "Enables building the unit tests" ON)
+option(ENABLE_TESTS "Enables building the unit tests" OFF)
 option(USE_SYSTEM_YAML "Uses a system version of yaml, if found")
 option(LTO_PCSX2_CORE "Enable LTO/IPO/LTCG on the subset of pcsx2 that benefits most from it but not anything else")
 
